pkg/api/api.go

```diff
@@ -11,14 +11,13 @@ const (
 
 // consts to maintain existing names of various sub-resources
 const (
-	OpenShiftConsoleName              = "openshift-console"
-	OpenShiftConsoleShortName         = "console"
+	OpenShiftConsoleName              = "console"
 	OpenShiftConsoleNamespace         = TargetNamespace
 	OpenShiftConsoleOperatorNamespace = "openshift-console-operator"
 	OpenShiftConsoleOperator          = "console-operator"
 	OpenShiftConsoleConfigMapName     = "console-config"
 	OpenShiftConsoleDeploymentName    = OpenShiftConsoleName
-	OpenShiftConsoleServiceName       = OpenShiftConsoleShortName
-	OpenShiftConsoleRouteName         = OpenShiftConsoleShortName
+	OpenShiftConsoleServiceName       = OpenShiftConsoleName
+	OpenShiftConsoleRouteName         = OpenShiftConsoleName
 	OAuthClientName                   = OpenShiftConsoleName
```

Comment:
I don't think we can change the name of the OAuth client. It's not our operator that creates it, right?

Reply:
OK, I see it's in the manifests, but we would need to update RBAC in that case.

Reply:
Ah, thanks for pointing that out.   Swinging back around and will test this one manually.  Applying all the manifests to my cluster & deploying it now to ensure things line up properly.